airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/TimeStringToIntegral.kt

```
@@ -0,0 +1,103 @@
+        return nanoOfDay / 1_000
+    }
+
+    private fun toMicrosOfDayWithoutTimezone(timeString: String): Long {
+        val time = LocalTime.parse(timeString, TIME_FORMATTER)
+        return time.toNanoOfDay() / 1_000
+    }
+
+    private fun toMicrosOfDay(timeString: String): Long {
+        return try {
+            toMicrosOfDayWithTimezone(timeString)
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)